examples/custom_checkers/r9901_impossible_condition.py

```diff
@@ -0,0 +1,28 @@
+def print_natural_number(x: int):
+    """
+    Print number x only if it's a natural number.
+    In CS natural number starts with 0.
+
+    Preconditions:
+        - x >= 0
+    """
+    if x < 0:
+        raise Exception("x is not a natural number")
+    print(x)
+
+
+def invalid_condition(x: bool):
```

Comment:
Don't use the term "invalid" here; "impossible" is fine.

Also, this example is quite good. The parallel example for the redundant condition checker would be `x or not x`; please add this example to that file.